src/query_graph.py

```diff
@@ -0,0 +1,67 @@
+from typing import get_type_hints, Dict, List
+
+from gremlin_python.structure.io import graphsonV3d0
+
+from src.graph_model import SecurityEvent, EventType, Dependency
+from src.graph_traversel import Traversel
+from src.parse_datetime import from_date_str
+from src.gremlin import GREMLIN
+
+# (fixme) traversel should start from ecosystem node
+def _query_template():
+    return '''
+        g.V().hasLabel('security_event').{security_event_query}.
```

Comment:
Similar to payload constant, you can think of moving labels, edge name etc to constant file.